README.md

```diff
@@ -22,7 +24,9 @@ The workflow for sale flagging is as follows:
 * Next, `glue/sales_val_flagging.py` flags all new, unflagged sales. This script is automated such that it runs on a schedule (e.g. monthly).
 * If an error occurs or we want to update the methodology on previously-flagged sales, `manual_flagging/manual_update.py` is used to select a subset of sales to re-flag. All sales have a version number that is incremented on update. When utilizing our sales views, we pull the flag data with the highest version value to keep it up-to-date.
```

Comment:
**issue:** I would reframe these as the 3 possible run modes for the sales val pipeline, based on the following conditions:

1. No sales are flagged --> initial run
2. Some sales are already flagged --> glue job
3. Sales must be re-flagged --> manual update

Then update the diagrams accordingly.